beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { TerritoryMapData } from "../../../interfaces";
+import WDCenter from "./WDCenter";
+import WDLabel from "./WDLabel";
+import WDUnitSlot from "./WDUnitSlot";
+
+interface WDTerritoryProps {
+  terr: TerritoryMapData;
```

Comment:
territoryMapData instead of terr? because two components have a prop that is terr could be confusing as they are different types